zio-http/src/main/scala/zhttp/http/Request.scala

```diff
@@ -96,18 +148,23 @@ object Request {
     url: URL = URL.root,
     headers: Headers = Headers.empty,
     remoteAddress: Option[InetAddress] = None,
-    data: HttpData = HttpData.Empty,
+    @unused data: HttpData = HttpData.Empty,
```

Comment:
smartly handle body based on the type

Reply:
I am guessing this still needs to be implemented?